### ISSUE TITLE
Implemention of ACLs for Subject, Body and Attachment

### ISSUE THREAD

**Issue by OlliW** [RotherOSS/otobo#1058]
For further possibilities for the ticket creation with the CustomerTicketMessage (instead the CustomerTicketProcess) it is necessary to hidde or show the Subject, Body and Attachment Fields.

**Comment by svenoe** [comment]
A package which allows making those fields optional, and or hiding them without any ACL interactions already exists. Integrating ACLs for standard fields are an additional step which will have to be looked into.

I will think about how to make the package available.

**Comment by svenoe** [comment]
Hm...a mention in a discussion is not noted in the issue apparently - I have a question concerning article creation which I wrote in: https://github.com/RotherOSS/otobo/discussions/1020#discussioncomment-833720

**Comment by OlliW** [comment]
Ok, do you have any progress on this point?

**Comment by svenoe** [comment]
I merged this into rel-10_1. We had some internal discussion on how to handle things, finally we decided to treat Subject, Body and Attachment together as "Article", this is where we are at. I kept the differentiation bewtween the three at some places however, to make it easier for packages to drop in here (actually maybe I should have also done that more in Core.AJAX.js).

In general we will (re)enable all other fields like Queue/Type/... because while some of them (the Queue e.g.) can be solved via Autoselect, this is not possible for all fields like SLA, or others. which are not mandatory for a ticket itself. Including them needs a little more work, though, as they have to be included in the convergence loops, as hiding them will delete their values.
